src/measures.cc

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+#include "measures.h"
+
+#include <absl/strings/string_view.h>
+#include <opencensus/stats/stats.h>
+
+namespace google {
+
+namespace {
+
+ABSL_CONST_INIT const absl::string_view kCount = "1";
```

Comment:
Maybe inline this on L39 for simplicity?

Reply:
there will be following PRs creating measure based on this `kCount`, so I will just leave this variable here.

Reply:
How about
```c++
constexpr char kCount[] = "1";
```

This doesn't have to be a `string_view`. Passing `constexpr char[]` to a function expecting `string_view` should be super cheap because the `strlen` should happen at compile time.

Reply:
As for `std::string`, I don't know about the metadata-agent style guide, but the Google style guide forbids static initialization of `std::string` because it's not trivially destructible.